src/style.css

```diff
@@ -1 +1,52 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Since the `html` and `body` tags are being styled with the same properties, you can combine them into a single rule to avoid redundancy.